### ISSUE TITLE
[Quest][Enchant] "Fiery Blaze Enchantments"

### ISSUE THREAD

**Issue by plznoz** [dalaranwow/dalaran-wow#3938]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The enchant given from this quest as a reward scales with spell damage, making it do stupid damage for certain classes (enh shaman and ret paladin).

**Expected behaviour**: Tell us what should happen instead
The enchant should not scale with spell damage.

**Steps to reproduce the problem**:

1. Do the quest "Fiery Blaze Enchantments" in the Badlands
2. Enchant your weapon on a class which has some spell power, for example Enh shaman/Ret paladin
3. See that the enchant does too much damage.

**Include proofs for this behaviour**
http://www.wowhead.com/item=5421/fiery-blaze-enchantment
In comments the user Skelic says spell damage has no effect in tbc, another user confirms on wotlk patch.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://wotlk.openwow.com/quest=706
http://wotlk.openwow.com/item=5421
http://wotlk.openwow.com/spell=6297

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by strobe-light** [comment]
This is also the case for a few other items, I have tested the following and they all scale with spellpower:
Skullflame Shield (Both the Drain Life and the Flamestrike)
Item ID: http://wotlk.openwow.com/item=1168
Drain Life ID: http://wotlk.openwow.com/spell=18817
Flamestrike ID: http://wotlk.openwow.com/spell=18816
Naglering
Item ID: http://wotlk.openwow.com/item=11669
Thorns ID: http://wotlk.openwow.com/spell=15438
Dragonbreath Chili - This is the one with the largest exploit potential as it can be used in raids without needing to use low level equipment
Item ID: http://wotlk.openwow.com/item=12217
Spell ID: http://wotlk.openwow.com/spell=15851 and http://wotlk.openwow.com/spell=15852

Proof (I just ran around in Icecrown with those items on and attacked some mobs):
![capture](https://cloud.githubusercontent.com/assets/15903359/20955134/6ed4af0e-bc8c-11e6-9a82-375603d88d38.JPG)

**Comment by Rushor** [comment]
These items/consumables will no longer scale with sp/ap

```sql
'Fiery Blaze'),
'Fiery Weapon - Fiery Weapon'),
'Demonslaying - Smite Demon'),
'Lifestealing - Life Steal'),
'Unholy Weapon - Unholy Curse'),
'Crusader - Holy Strength'),
'Battlemaster - Battlemaster'),
'Icebreaker - Icebreaker'),
'Lifeward - Lifeward'),
'Deathfrost - Deathfrost'),
'Blade Warding - Blade Warding'),
'Blood Draining - Blood Reserve');
'Blazefury Medallion & Fiery Retributer (Fire Strike)'),
'Fiery Plate Gauntlets (Fire Strike)'),
'Gauntlets of the Sea (Heal)'),
'Storm Gauntlets (Lightning Strike)'),
'Skullforge Reaver - Skullforge Brand'),
'Freezing Band (Freeze)'),
'Thunderfury'),
'Heart of Wyrmthalak (Flame Lash)'),
'Paladin T3 (8)'),
'Warlock - Siphon Essence - T6 2P proc'),
'Crystal Spire of Karabor - heal effect'),
'Brunnhildar weapons (Chilling Blow)'),
'Living Ice Crystals - heal effect'),
'Pillar of Flame (Normal)'),
'Pillar of Flame (Heroic)'),
'Drain Life'),
'Drain Life'),
'Flamestrike'),
'Thorns');
 'Consumable - Flamecap (Flamecap Fire)'),
 'Poison - Bloodboil Poison'),
 'Consumable - Stormchops (Lightning Zap)'),
 'Consumable - Stormchops (Lightning Zap)'),
 'Consumable - Dragonbreath Chili');
```

If you find more, let me know and i will add them to the whitelist.

DB/Spell: Update Spell damage item procs scalling with spellpower 

* closes https://github.com/dalaranwow/dalaran-wow/issues/3938
* commit: 9de3832360d8bd5432152d8088e10dfa2720c4dc

**Comment by Darthblaider** [comment]
Naglering and skullflame shield ARE supposed to scale with spellpower and that much is blizzlike

**Comment by Edzero** [comment]
Proof for Skullflame:
http://www.wowhead.com/item=1168/skullflame-shield#comments

> Scales with spell power, just tested this out on my 80 ret paladin with 1600 spell power, the drain life procced for 1586 shadow damage, healing me for that amount, not sure if this would be more useful than a 245+ shield I have from raiding for soloing low dungeons.

Also, the fire damage it deals does not scale with spell power, it is a static amount.

> Picked this up on my shaman for fun to try out the proc:
The fire proc doesn't scale with SP, the lifesteal one does. Lifesteal scales 100% with spellpower.

However, for some reason the life stealing proc won't proc without a block, I pulled the entire stockade, put my back to them and only got fire procs for 10 minutes straight. I turned towards them, and they started dropping like flies from the life stealing, all of them were dead in the first minute.



The only relevant comments I could find from retail regarding Naglering are these:

http://www.wowhead.com/item=11669/naglering#comments

Svstoned mentions scaling with spellpower on other similar items but not the Naglering. 

Holymonkey said:

> This does way more than 3 damage a hit I was getting 48s per hit at lvl 54...does it stack with anything else? Because I don't have any other items that do damage on hit.

Which was downvoted and the following reply was made:

> You're seeing the effect of a druid casting thorns on you. You can't see the 3 damage usually esp in aoe packs of Naglering because the thorns will be all over, but it -is- proccing, just not visible. Source: Had the same thing happening to me for days then had an epiphany.

From Soulcorruptr:

> Appears to be back to a state of working today. Damage is being reported as "Thorns" reflecting the 3 damage. Funnily enough, sometimes it reflects 4 damage.
